spec/services/activity_defaults_spec.rb

```diff
@@ -75,7 +114,7 @@
       end
 
       it "sets the transparency identifier" do
```

Comment:
If we're going to keep this style of `it`, might be worth updating this test description to `it "sets the transparency identifier to nil"` here (and below) so it's a bit more obvious it's a special case.